test-cases/nemesis/longevity-5gb-1h-SlaDecreaseSharesDuringLoad.yaml

```diff
@@ -1,22 +1,23 @@
-test_duration: 90
+test_duration: 140
 
-prepare_write_cmd:  "cassandra-stress write cl=QUORUM n=5048570 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=80 -pop seq=1..5048570 -col 'n=FIXED(8) size=FIXED(128)' -log interval=5"
+prepare_write_cmd:  "cassandra-stress write cl=QUORUM n=5048570 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) -mode cql3 native -rate threads=800 -pop seq=1..5048570 -log interval=5"
```

Comment:
@juliayakovlev 

when backporting I've notice the -schema arguments in all this file are missing a closing `'`